### ISSUE TITLE
Package react dependecy not compatible with react 19 apparently

### ISSUE THREAD

**Issue by mambucodev** [vercel/analytics#148]
I just updated to nextjs 15 with the new codemod, and it proposed me to update to react 19. I said yes, now if I install stuff npm has issues with react, coming from @vercel/analytics.

```
❯ npm i
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @vercel/analytics@1.3.1
npm error Found: react@19.0.0-rc-65a56d0e-20241020
npm error node_modules/react
npm error   react@"19.0.0-rc-65a56d0e-20241020" from the root project
npm error   peer react@"^16.8 || ^17.0 || ^18.0 || ^19.0 || ^19.0.0-rc" from @radix-ui/react-alert-dialog@1.1.2
npm error   node_modules/@radix-ui/react-alert-dialog
npm error     @radix-ui/react-alert-dialog@"^1.1.2" from the root project
npm error   33 more (@radix-ui/react-arrow, @radix-ui/react-aspect-ratio, ...)
npm error
npm error Could not resolve dependency:
npm error peerOptional react@"^18 || ^19" from @vercel/analytics@1.3.1
npm error node_modules/@vercel/analytics
npm error   @vercel/analytics@"^1.3.1" from the root project
npm error
npm error Conflicting peer dependency: react@18.3.1
npm error node_modules/react
npm error   peerOptional react@"^18 || ^19" from @vercel/analytics@1.3.1
npm error   node_modules/@vercel/analytics
npm error     @vercel/analytics@"^1.3.1" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /home/mambuco/.npm/_logs/2024-10-22T17_16_36_558Z-eresolve-report.txt
npm error A complete log of this run can be found in: /home/mambuco/.npm/_logs/2024-10-22T17_16_36_558Z-debug-0.log
```

**Comment by mambucodev** [comment]
Dependecies:
```
  "dependencies": {
    "@radix-ui/react-alert-dialog": "^1.1.2",
    "@radix-ui/react-aspect-ratio": "^1.1.0",
    "@radix-ui/react-label": "^2.1.0",
    "@radix-ui/react-popover": "^1.1.2",
    "@radix-ui/react-select": "^2.1.2",
    "@radix-ui/react-slot": "^1.1.0",
    "@supabase/supabase-js": "^2.45.6",
    "@vercel/analytics": "^1.3.1",
    "class-variance-authority": "^0.7.0",
    "clsx": "^2.1.1",
    "lucide-react": "^0.453.0",
    "next": "15.0.0",
    "next-themes": "^0.3.0",
    "react": "19.0.0-rc-65a56d0e-20241020",
    "react-dom": "19.0.0-rc-65a56d0e-20241020",
    "tailwind-merge": "^2.5.4",
    "tailwindcss-animate": "^1.0.7"
  },
  "devDependencies": {
    "@types/node": "^22",
    "@types/react": "npm:types-react@19.0.0-rc.1",
    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
    "eslint": "^8",
    "eslint-config-next": "15.0.0",
    "postcss": "^8",
    "shadcn": "^2.1.0",
    "tailwindcss": "^3.4.14",
    "typescript": "^5"
  },
  "overrides": {
    "@types/react": "npm:types-react@19.0.0-rc.1",
    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1"
  }
  ```

**Comment by rnkdsh** [comment]
+1

**Comment by feugy** [comment]
Thanks for letting us now!
The new version has just been published: [`@vercel/analytics@1.3.2`](https://www.npmjs.com/package/@vercel/analytics/v/1.3.2)
